app/models/concerns/queue_it/queable.rb

```diff
@@ -58,12 +58,21 @@ def delete_queue_nodes
       queue.nodes.delete_all
     end
 
-    def delete_node(position)
-      raise Error('The queue is empty') if queue.nodes.zero?
-      raise Error('position out of length or invalid') if position > queue.nodes.length - 1
-      # SIEMPRE TENEMOS 3 CASOS A NIVEL DE LISTA Y NIVEL DE POSICIONES:
-      # A NIVEL DE LISTAS: LA LISTA TIENEN 1, 2 O MÁS NODOS
-      # A NIVEL POSICIÓN ES LA CABEZA, LA COLA O CUALQUIER OTRO
+    def remove_from_queue(nodable)
+      return if local_queue.empty? || local_queue.nodes.where(nodable: nodable).empty?
+
+      ActiveRecord::Base.transaction do
+        local_queue.lock!
+        local_queue.nodes.where(nodable: nodable).each do |node|
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Rails/FindEach: Use find_each instead of each.